content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst

```diff
@@ -0,0 +1,189 @@
+======================
+Requests for quotation
+======================
+
+.. _purchase/manage_deals/rfq:
+
+.. |PO| replace:: :abbr:`PO (Purchase Order)`
+.. |POs| replace:: :abbr:`POs (Purchase Orders)`
+.. |RFQ| replace:: :abbr:`RFQ (Request for Quotation)`
+.. |RFQs| replace:: :abbr:`RFQs (Requests for Quotation)`
+
+Odoo's requests for quotation (RFQs) feature in the **Purchase** app standardizes ordering products
+from multiple vendors with varying prices and delivery times.
+
+|RFQs| are documents companies send to vendors requesting product pricing. In Odoo, once the vendor
+approves the |RFQ|, the purchase order (PO) is confirmed to align on lead times and pricing.
+
+Configuration
+=============
+
+Product
+-------
+
+To auto-populate product information and prices on an |RFQ|, configure products by going to
+:menuselection:`Purchase app --> Products --> Products`. Select an existing product, or create a
+new one by selecting :guilabel:`New`. Doing so opens the product form, where various sales and
+purchasing data can be configured.
+
+To configure purchasable products, tick the :guilabel:`Can be purchased` checkbox, under the product
+name. Then, go to the :guilabel:`Inventory` tab, and enable the :guilabel:`Buy` route.
+
+.. image:: rfq/product-vendor-pricelist-config.png
+   :align: center
+   :alt: Required configuration for purchasable products.
+
+Vendor pricelist
+----------------
+
+In the :guilabel:`Purchase` tab of the product form, input the vendor and their price, to have this
+information auto-populate on an |RFQ| each time the product is listed.
+
+Default columns include :guilabel:`Quantity`, :guilabel:`Price`, and :guilabel:`Delivery Lead Time`,
+but other columns like, :guilabel:`Product Variant` or :guilabel:`Discounts`, can also be enabled.
+
+To enable or disable columns, click the :icon:`oi-settings-adjust` :guilabel:`(settings)` icon on
+the right side of the header row to reveal a drop-down menu of additional columns that can be added
+(or removed) from the :guilabel:`Purchase` tab.
```

Comment:
Small change to update the icons label name, this icon is usually referred to as "additional options" :)
```suggestion
To enable or disable columns, click the :icon:`oi-settings-adjust` :guilabel:`(additional options)`
icon on the right side of the header row to reveal a drop-down menu of additional columns that can
be added (or removed) from the :guilabel:`Purchase` tab.
```